packages/azure-kusto-ingest/src/fileDescriptor.browser.ts

```diff
@@ -50,4 +56,14 @@ export class FileDescriptor extends AbstractDescriptor implements FileDescriptor
     getCompressionSuffix() {
         return this.compressionType ? `.${this.compressionType}` : ".gz";
     }
+
+    private _calculateSize(modifier: number = 1): void {
```

Comment:
Why is this a fileDescriptor thing to do ? its relevant to all other descriptors ..
Its easier when we align all this king of logic to our other SDKs - this way we can make sure we dont have loose ends